### ISSUE TITLE
At node start, first fee estimate always fails and defaults to default fee rate

### ISSUE THREAD

**Issue by SimonVrouwe** [ElementsProject/lightning#1698]
### Issue and Steps to Reproduce
Every time I start the node, the log shows `No fee estimate for Immediate: basing on default fee rate` Was wondering if that is normal and is there a way to prevent this. I am working on exponential smoothing of fee estimates and because of this initial defaulting of the feerate to 40000 sat/kw it takes a while to smooth back to the _real_ fee estimates.

It only happens at startup.

```
2018-07-14T09:08:52.769Z lightningd(8395): No fee estimate for Immediate: basing on default fee rate
2018-07-14T09:08:52.601Z lightningd(8395): No fee estimate for Immediate: basing on default fee rate
2018-07-14T09:08:52.601Z lightningd(8395): No fee estimate for Slow: basing on default fee rate
2018-07-14T09:08:52.618Z lightning_gossipd(8404): Connected out for 02fd98ebd4cbedd1317d629d62d192cf943f0134c61464de48f1e232d861de4ef9
2018-07-14T09:08:52.677Z lightningd(8395): Adding block 531848: 0000000000000000002296ac6ae8df440d19039abbfeca61c3adf903d8b9523b
2018-07-14T09:08:52.767Z lightning_gossipd(8404): peer_out WIRE_INIT
2018-07-14T09:08:52.767Z lightningd(8395): lightning_channeld-02977901c53b5299c7641acf11d49a46f0957e9e3f8191e6171d1222486d872317 chan #5: init LOCAL: remote_per_commit = 03df1a4c265ae43e9c1a828b
be2dba5db5f9f857ed45e01c8b4ffceb866349ace8, old_remote_per_commit = 021dfd2e14f52a4a6eff41d89b6c41beffa44733c95dcd1d279930722eccbdcc47 next_idx_local = 118 next_idx_remote = 118 revocations_rece
ived = 117 feerates 1569/1569 (range 5000-400000)
2018-07-14T09:08:52.767Z lightningd(8395): lightning_channeld-02977901c53b5299c7641acf11d49a46f0957e9e3f8191e6171d1222486d872317 chan #5: peer_out WIRE_CHANNEL_REESTABLISH
2018-07-14T09:08:52.769Z lightningd(8395): No fee estimate for Immediate: basing on default fee rate
2018-07-14T09:08:52.769Z lightningd(8395): Immediate feerate 8486 (was 0)
2018-07-14T09:08:52.769Z lightningd(8395): ...feerate 1569 hit floor 8486
2018-07-14T09:08:52.769Z lightningd(8395): No fee estimate for Normal: basing on Immediate rate
2018-07-14T09:08:52.769Z lightningd(8395): Normal feerate 1202 (was 0)
2018-07-14T09:08:52.769Z lightningd(8395): ...feerate 535 hit floor 1202
2018-07-14T09:08:52.769Z lightningd(8395): No fee estimate for Slow: basing on Immediate rate
2018-07-14T09:08:52.769Z lightningd(8395): No fee estimate for Slow: basing on Normal rate
2018-07-14T09:08:52.769Z lightningd(8395): Slow feerate 327 (was 0)
2018-07-14T09:08:52.769Z lightningd(8395): ...feerate 267 hit floor 327
```

### `getinfo` output
```
{
  "id": "0323dad2e405f615b79ab93e133c709e18456c1c742aca2308180c767b9c0b8b60", 
  "alias": "LookBusy2", 
  "color": "0323da", 
  "port": 9735, 
  "address": [
  ], 
  "binding": [
    {
      "type": "ipv6", 
      "address": "::", 
      "port": 9735
    }
  ], 
  "version": "v0.6-64-g14c6310", 
  "blockheight": 531888, 
  "network": "bitcoin"
}
```
running `Bitcoin Core RPC client version v0.16.1
` on same PC

**Comment by SimonVrouwe** [comment]
I think this is what happens and it has nothing to do with bitcoind or rpc and looks normal behaviour. At the first _actual_ `feerate` poll, it reads the previous `feerate` here:
https://github.com/ElementsProject/lightning/blob/19a0b78fec68cf8cb382f065efc7a720108bf6c1/lightningd/chaintopology.c#L250
Which was initialised to zero, here:
https://github.com/ElementsProject/lightning/blob/19a0b78fec68cf8cb382f065efc7a720108bf6c1/lightningd/chaintopology.c#L691
and here:
https://github.com/ElementsProject/lightning/blob/19a0b78fec68cf8cb382f065efc7a720108bf6c1/lightningd/chaintopology.c#L705
and which therefore defaults to `topo->default_fee_rate = 40000` for `FEERATE_IMMEDIATE` and halve of that and a quarter of that for `FEERATE_NORMAL` and `FEERATE_SLOW` respectively:
https://github.com/ElementsProject/lightning/blob/19a0b78fec68cf8cb382f065efc7a720108bf6c1/lightningd/chaintopology.c#L565

But after f2ffb6d03eb769b7327f322dcaad27c1a75221e2 this can cause unilateral close of channels when an `update_fee` message is send to peers with a fee rate set to the smoothed values of the `default feerate`.

**Comment by SimonVrouwe** [comment]
Fixed by #1752
